tests/eas/load_tracking.py

```diff
@@ -537,6 +540,367 @@ def test_load_avg_behaviour(self, experiment, tasks):
         """
         return self._test_behaviour(experiment, tasks, 'load_avg')
 
+class _CPUMigrationBase(LisaTest):
+    """Base class for shared functionality of tasks migration tests"""
+    test_conf = {
+        'tools'    : [ 'rt-app' ],
+        'ftrace' : {
+            'events' : [
+                'sched_switch',
+                'sched_load_cfs_rq',
+            ],
+        },
+        'modules': ['cpufreq', 'cgroups'],
+    }
+
+    # Allowed error margin
+    allowed_util_margin = UTIL_SCALE * 0.02
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(_CPUMigrationBase, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def get_wload(cls, test_env, tasks, workload_name):
+        """Create a specification for a rt-app workload from a list of tasks
+        :param test_env: Environment of the test
+        :type test_env: TestEnv
+
+        :param tasks: lists of tasks
+        :type tasks:[RTATask]
+
+        :param workload_name: name of the workload
+        :type workload_name: str
+
+        :return the workload specification
+        """
+        rtapp = RTA(test_env.target, workload_name,
+            calibration=test_env.calibration())
+        rtapp.conf(kind='profile',
+            params={'task{}'.format(i): task.get() for i, task in enumerate(tasks)},
+            run_dir=Executor.get_run_dir(test_env.target))
+        return {
+            'migration': {
+                'type' : 'rt-app',
+                'conf' : {
+                    'class' : 'custom',
+                    'json' : rtapp.json,
+                    'prefix' : 'mig_test',
+                },
+            }
+        }
+
+    def _get_one_task(self, tasks):
```

Comment:
Mmm... would say either `_get_first_task` or `_get_one_task(self, tasks, id)`... and I would slightly prefer the second.

Reply:
Done: _get_one_task(self, tasks, id)